.circleci/config.yml

```diff
@@ -36,59 +41,61 @@ jobs:
     executor:
       name: node/default
       tag: '10.16'
-    environment:
-      EXCHANGE_DOWNLOAD: true
     steps:
-      - run: echo Publish commerce_sdk to npm "$EXCHANGE_DOWNLOAD"
       - checkout
-        # Add read/write ssh key's finger print in order to push the tags
-      - add_ssh_keys:
-          fingerprints:
-            - "78:6d:b4:67:eb:35:d4:a0:63:af:c2:32:f1:94:9a:5f"
       - node/with-cache:
           steps:
             - run: npm install
       - run: npm run build
-      - run: echo -e "$GPG_KEY" | gpg --import
-      - run: git config --global user.email "$GITHUB_PUBLISH_EMAIL"
-      - run: git config --global user.name "$GITHUB_PUBLISH_USER"
-      - run: git config --global user.signingkey $GITHUB_GPG_SIGN_KEY_ID
-      - run: git config --global commit.gpgsign true
       - run:
           name: Create npmrc file to authenticate with npm registry.
           command: echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc
       - run: npm run publish
-      - run: sudo npm install -g @octokit/rest
-      - run: node .circleci/create_pr.js
-
 workflows:
   version: 2
   build-and-test:
     jobs:
       - build:
           filters:
-            branches:
-              ignore:
-                - /^release.*/
+            tags:
+              ignore: /^v.*/
       - build-12:
-        filters:
-          branches:
-            ignore:
-              - /^release.*/
-  build-test-and-deploy:
+          filters:
+            tags:
+              ignore: /^v.*/
+  build-test-deploy:
```

Comment:
This workflow will get executed for every tagged commit, 

For example one of the checks in this PR https://circleci.com/gh/SalesforceCommerceCloud/commerce-sdk/937?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link actually published the package from an unapproved branch...

Reply:
This wasn't unintentional.  This was an intentional test of the deployment path.  It only runs this on tagged commits

Reply:
The problem is unless we publish, PRs containing tagged commits cannot be merged to master.. Is this what we really want?

Reply:
Why can't they be merged to master?

Reply:
As discussed in our standup, we are not doing annotated commits. we can merge to master.. This comment can be resolved